package.json

```diff
@@ -50,7 +50,7 @@
     "cozy-client": "24.0.0",
     "cozy-scripts": "5.13.0",
     "cozy-ui": "51.5.1",
-    "date-fns": "^1.30.1",
+    "date-fns": "2.27.0",
```

Comment:
can you run a `yarn why date-fns` please? Do we now have at least two versions of this lib? Can you check the bundle size before and after?

Reply:
What is the gain you were hoping for with this upgrade?

Reply:
Le gain était d'avoir une lib moins obsolète, mais comme on l'utilise aussi dans cozy-ui et cozy-bar, on va rester sur l'ancienne version ce sera plus simple